packages/core/src/test/techdebt.test.ts

```diff
@@ -18,14 +18,10 @@ describe('tech debt', function () {
 
     it('vscode minimum version', async function () {
         const minVscode = env.getMinVscodeVersion()
+        assert.ok(semver.lt(minVscode, '1.84.0'))
 
         assert.ok(
-            semver.lt(minVscode, '1.75.0'),
-            'remove filesystemUtilities.findFile(), use vscode.workspace.findFiles() instead (after Cloud9 VFS fixes bug)'
-        )
-
-        assert.ok(
-            semver.lt(minVscode, '1.75.0'),
+            semver.lt(minVscode, '1.93.0'),
             'remove AsyncLocalStorage polyfill used in `spans.ts` if Cloud9 is on node 14+'
```

Comment:
I think we actually need the polyfill for web-mode, right @nkomonen-amazon  ?

Reply:
Yeah I'm pretty sure that is correct. `AsyncLocalStorage` is node specific and I think that in web we are falling back to our polyfill here: https://github.com/aws/aws-toolkit-vscode/blob/d6012a50c6632b8761a866ec23874182c59a1a3c/packages/core/src/shared/telemetry/spans.ts#L29

If we get rid of this then telemetry wont work

Reply:
@Hweinstock so this test can be deleted